index.html

```diff
@@ -859,13 +858,26 @@ <h3>
           application, it SHOULD treat that manifest as a description of a
           distinct application, even if it is served from the same URL as that
           of another application. When the user agent sees a manifest where
-          |manifest|["id"] is [=url/equal=] with [=URL serializer/exclude
-          fragment|exclude fragment true=] to the [=identity=] of an
-          already-installed application, it SHOULD be used as a signal that
+          |manifest|["id"] is [=url/equal=] (OPTIONAL: with
+          [=URL/equals/exclude fragments=] set to true) to the [=identity=] of
+          an already-installed application, it SHOULD be used as a signal that
           this manifest is a replacement for the already-installed
           application's manifest, and not a distinct application, even if it is
           served from a different URL than the one seen previously.
         </p>
+        <p class="note" title="Exclude fragments for good measure">
+          Since the [=process the id member|processing algorithm=] removes the
+          [=URL/fragment=] from the <code>[=manifest/id=]</code> member, it is not
+          strictly necessary to [=URL/equals/exclude fragments=] when checking
+          for a matching application. However, since old versions of this spec
+          (and, possibly, old user agents) did not remove the [=URL/fragment=]
+          from the [=URL=] at parse time, and relied only on
+          [=URL/equals/exclude fragments|excluding fragments=] during
+          comparisons, there may be historical app data with [=URL/fragments=]
+          in the <code>[=manifest/id=]</code>. Therefore, it is best practice for user
+          agents to [=URL/equals/exclude fragments=] even when comparing two
+          [=URLs=] that should not have any fragments.
```

Comment:
Removing the use of RFC2119 keywords: 

```suggestion
          for a matching application. However, since old versions of this spec
          (and, possibly, old user agents) did not remove the [=URL/fragment=]
          from the [=URL=] at parse time, and relied only on
          [=URL/equals/exclude fragments|excluding fragments=] during
          comparisons, historical app data could include [=URL/fragments=]
          in the <code>[=manifest/id=]</code>. Therefore, it is best practice for user
          agents to [=URL/equals/exclude fragments=] even when comparing two
          [=URLs=] that typically lack fragments.
```

Reply:
Thanks, good catch! I rephrased these a bit but got rid of the RFC words.